components/search/SearchSuggestion.vue

```diff
@@ -299,6 +298,20 @@ export default class SearchSuggestion extends mixins(PrefixMixin) {
     }
   }
 
+  public updateSearchUrl() {
+    if (this.name) {
+      this.$router
+        .replace({
+          path: String(this.$route.path),
+          query: {
+            page: '1',
+            search: this.name || undefined,
```

Comment:
```suggestion
            search: this.name,
```
I don't think it would be undefined if we check `this.name` beforehand